spaceship/docs/iTunesConnect.md

```diff
@@ -296,22 +296,30 @@ app.add_external_tester!(email: "github@krausefx.com", first_name: "Felix", last
 Right now, `spaceship` can't modify or create internal testers.
 
 # Load all sandbox testers
+```ruby
 testers = Spaceship::Tunes::SandboxTester.all
+```
 
 # Create a sandbox tester
```

Comment:
Why are those header 1s? That looks kind of weird. Either that should stay in the code (being a comment) or not be header 1